test-junit-engine/src/main/scala/zio/test/junit/ReflectionUtils.scala

```diff
@@ -0,0 +1,23 @@
+package zio.test.junit
+
+import scala.util.Try
+
+object ReflectionUtils {
```

Comment:
I think we can make this object `private` right?

Reply:
Right